jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/BottomSheetsTests.java

```diff
@@ -102,19 +102,20 @@ public void playerBottomSheetTest() {
     }
 
     @Test
-    public void listBottomSheetTest(){
+    public void listBottomSheetTest() {
         listBottomSheet.is().hidden();
 
         listBottomSheetButton.click();
         listBottomSheet.is().displayed();
-        listBottomSheet.openInOptionsList().get(1).find(".v-list-item__title").has().text("Keep");
-        listBottomSheet.openInOptionsList().get(1).find("img").click();
+        listBottomSheet.getOptionTitle(1, "Keep");
+        listBottomSheet.clickOption(1);
         listBottomSheet.is().hidden();
 
+
```

Comment:
what's the reason to recheck element one more time? You can check all available options or first and last points as minimum

Reply:
Thanks for your help, I will check first and last points